libportal/meson.build

```diff
@@ -63,6 +63,12 @@ libportal = library('portal',
                     install: true,
                     dependencies: [gio_dep, gio_unix_dep])
 
+pkgconfig.generate(libportal,
+  description: 'Portal API wrappers',
+  name: 'libportal',
+  requires: [ gio_dep ],
```

Comment:
That should be `gio_unix_dep` otherwise you're changing the `Requires` from `gio-unix-2.0` to `gio-2.0`. I don't know whether that's correct, but if it is, it needs to happen in a separate commit.

Reply:
Should I fix it in the old `.pc.in` template explicitly? It is not really possible to re-create the previous incorrect file using the Meson’s `pkgconfig` module since it correctly adds `library`’s `dependencies` to `Requires.private` (https://mesonbuild.com/Pkgconfig-module.html#implicit-dependencies).

Reply:
> Should I fix it in the old .pc.in template explicitly?

That would be best, thanks.

Reply:
Done.